code/modules/mob/living/carbon/human/say.dm

```diff
@@ -6,8 +6,8 @@
 		. = ..()
 
 /mob/living/carbon/human/GetVoice()
-	if(istype(wear_mask, /obj/item/clothing/mask/chameleon))
-		var/obj/item/clothing/mask/chameleon/V = wear_mask
+	if(istype(wear_mask, /obj/item/clothing/mask))
```

Comment:
Just make the mask chameleon subtype but rip out cool stuff from it

Reply:
I'm not sure that would be better.

Reply:
That would mean this code would run only for 2 masks instead of all of them